src/middlewares/getOneArticle.js

```diff
@@ -0,0 +1,36 @@
+import models from '../api/models';
+import statusCode from '../helpers/constants/status.codes';
+
+const { Article } = models;
+/**
+ * containing all user's model controllers (signup, login)
```

Comment:
Please, change this description @MCFrank16

Reply:
Didn't I change it??? I am on it now